processing/src/main/java/io/druid/query/aggregation/hyperloglog/HyperLogLogCollector.java

```diff
@@ -640,7 +640,7 @@ private short addNibbleRegister(short bucket, byte positionOf1)
    * @param offsetDiff The difference in offset between the byteToAdd and the current HyperLogLogCollector
    * @param byteToAdd  The byte to merge into the current HyperLogLogCollector
    */
-  private static int mergeAndStoreByteRegister(
+  private static short mergeAndStoreByteRegister(
```

Comment:
I wonder if this has any performance implications, since this method is called very frequently